tests/TransportTestCase.php

```diff
@@ -104,10 +104,15 @@ public function testFollowLocation()
                 'pattern' => 'array_merge'
             ]);
 
-        $response = $request->setOptions([
-            'followLocation' => false,
-        ])->send();
-        $this->assertEquals('302', $response->statusCode);
+        /**
+         * HHVM does not take `follow_location` option
+         */
+        if (!defined('HHVM_VERSION')) {
```

Comment:
It seems we have to remove default set of `followLocation`, since ut can silently break at some environment.